docs/content/faq/how-can-my-containers-talk-to-each-other.md

```diff
@@ -0,0 +1,5 @@
+# How can my containers talk to each other?
+
+Flynn provides DNS resolution for all apps that it runs. By default, cloud installations will use flynnhub.com, and local developer environments will use dev.localflynn.com. The pattern is `<app name>.yourdomain.com`, e.g. if your cluster uses `emyx.flynnhub.com`, and your app is called `blog`, any of your other apps will be able to access it at `blog.emyx.flynnhub.com`. For web processes, Flynn also provides `<app name>-web.discoverd`, e.g. `blog-web.discoverd`.
```

Comment:
Containers should probably just talk to each other using the internal discoverd hostnames, rather than the external `xxx.$CLUSTER_DOMAIN` routes.